start.js

```diff
@@ -12,14 +12,16 @@ const pump = require('pump')
 const isDocker = require('is-docker')
 const listenAddressDocker = '0.0.0.0'
 const watch = require('./lib/watch')
+const tsWatch = require('./lib/watch/tsc-watcher')
```

Comment:
can you load this only on demand? We are loading the entirety of typescript even if it's not a ts project.